OsmAnd-api/src/net/osmand/aidlapi/map/APosition.java

```diff
@@ -0,0 +1,138 @@
+package net.osmand.aidlapi.map;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class APosition implements Parcelable {
```

Comment:
1. APosition > ALocation
2. ALocation immutable
3. ALocation extends AidlParams
4. positionType > locationType and it should be explained in the method's description

Reply:
Done in latest commit